demos/nginx/testdata/clusterlink/peer-relay.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: clusterlink.net/v1alpha1
+kind: Peer
+metadata:
+  name: client
```

Comment:
IIUC, the flow is `client` imports from `relay`, imports from `server`.
If that's correct, we don't really need a peer representing the client (no other peer needs to connect to its gateways).

While it is used on the `client` cluster, I think the Peer `name` and `gateways` should be aligned with representing `relay`, not `client`